test/controllers/admin_authentication_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class AdminAuthenticationControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
```

Comment:
@abapolov we aren't using MiniTest in the project anymore. Please remove this file, it doesn't make sense.